config.json

```diff
@@ -18,447 +18,14 @@
-        "prerequisites": [
-          "variables"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "range-iteration",
-        "uuid": null,
-        "concepts": [
-          "range-iteration"
-        ],
-        "prerequisites": [
-          "iteration",
-          "maps",
-          "slices"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "reflection",
-        "uuid": null,
-        "concepts": [
-          "reflection"
-        ],
-        "prerequisites": [
-          "interfaces",
-          "empty-interface"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "select",
-        "uuid": null,
-        "concepts": [
-          "select"
-        ],
-        "prerequisites": [
-          "channels"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "slices",
-        "uuid": "8bc0c6f6-1891-4688-9ed7-a63e3702f655",
-        "concepts": [
-          "slices"
-        ],
-        "prerequisites": [
-          "iteration",
-          "numbers"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "strings",
-        "uuid": "e09523d2-c91f-11ea-87d0-0242ac130003",
-        "concepts": [
-          "strings"
-        ],
-        "prerequisites": [
-          "variables"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "strings-package",
-        "uuid": "b97a271c-4036-4ea8-8c6a-094715a48627",
-        "concepts": [
-          "strings-package"
-        ],
-        "prerequisites": [
-          "slices"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "structs",
-        "uuid": "2508aa96-e4e9-444e-9e6b-8b748c4f9305",
-        "concepts": [
-          "structs"
-        ],
-        "prerequisites": [
-          "numbers",
-          "conditionals"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "time",
-        "uuid": "1c1414d4-3688-4e9d-bc38-908eb7adf970",
-        "concepts": [
-          "time"
-        ],
-        "prerequisites": [
-          "numbers",
-          "strings"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "type-assertion",
-        "uuid": null,
-        "concepts": [
-          "type-assertion"
-        ],
-        "prerequisites": [
-          "interfaces"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "type-conversion",
-        "uuid": null,
-        "concepts": [
-          "type-conversion"
-        ],
-        "prerequisites": [
-          "strings",
-          "numbers"
-        ],
-        "status": "wip"
-      },
-      {
-        "slug": "zero-value",
-        "uuid": "7b265cc8-8f88-4de1-b653-10a384e6c7e2",
-        "concepts": [
-          "zero-value",
-          "nil"
-        ],
-        "prerequisites": [
-          "functions",
-          "numbers",
-          "strings",
-          "pointers",
-          "booleans"
-        ],
-        "status": "wip"
+        "status": "active"
```

Comment:
I don't know if you already knew, but if the status is set to `actice`, you can also omit the status entirely.

Reply:
I just happened  to read that last night, thanks! I'll leave them in for now, until we have some samples with other states in the config file that people can refer to.